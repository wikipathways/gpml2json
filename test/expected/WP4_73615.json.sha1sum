6f34afd1d14566a668e66661b2af7961b8ad0223 ?-
