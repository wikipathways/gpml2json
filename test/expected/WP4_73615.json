{"pathway":{"@context":["https://cdn.rawgit.com/wikipathways/WpVocabularies/7a46a05/contexts/pvjs.jsonld"],"backgroundColor":"white","borderWidth":0,"color":"black","contains":["c5ad2","idd31b36db","id1b82938b","iddb017374","id476a1b22","ida7510884","id1006ca65","id3a237139","id55ba47f0","idd937525f","ida549ffdf","ab8f5","b621a","a2472","dce23","a489f","daafb","c7084","d8bae","da9a8","f376f","d32e4","f917f","e9680","aecda","b07f7","f9e19","d6c00"],"drawAs":"rect","gpmlElementName":"Pathway","height":540,"kaavioType":"Group","name":"Sandbox Pathway","padding":5,"type":["Pathway","http://identifiers.org/pw/PW:0001310","http://identifiers.org/pw/PW:0000125"],"width":768.6990077177508,"x":0,"y":0,"zIndex":0,"fontSize":12,"fontWeight":"bold","textAlign":"left","verticalAlign":"top","comments":[{"source":"WikiPathways-description","content":"This is just a test pathway with a variety of object types including datanodes which can be connected to database entries (double-click on them when in edit mode). Feel free to edit anything on this page."},{"source":"BotTest","content":"20140201000004"}],"lastModified":"1/28/2016","organism":"Homo sapiens","license":"CC BY 2.0","standardName":"Sandbox Pathway","displayName":"Sandbox Pathway","textContent":"Name: Sandbox Pathway\nLicense: CC BY 2.0\nLast Modified: 1/28/2016\nOrganism: Homo sapiens","isSimilarTo":"http://wikipathways.org/index.php/Special:SearchPathways?query=Sandbox%20Pathway&species=Homo%20sapiens&doSearch=1","id":"hash://sha256/df79663a107b9ef0cd087bff6442865361cb85f22e3eed277fe0043409de5349"},"entityMap":{"a489f":{"textContent":"Substrate","id":"a489f","wpType":"Metabolite","x":213.88250118128843,"y":170.6562713288181,"width":80,"height":20,"zIndex":32768,"fontSize":10,"verticalAlign":"middle","color":"#0000ff","textAlign":"center","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","Metabolite"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"c5ad2":{"id":"c5ad2","cellularComponent":"Cell","x":130.91915836101896,"y":83.87264673311188,"width":607.7798493567318,"height":426.1273532668881,"zIndex":-2,"fontSize":10,"verticalAlign":"middle","drawAs":"rect","borderRadius":15,"borderWidth":3,"color":"#c0c0c0","rotation":0,"textAlign":"center","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode","PhysicalEntity","CellularComponent","Cell"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"a2472":{"textContent":"Nucleus","id":"a2472","x":536.7441860465123,"y":334.186046511628,"width":98.91472868217053,"height":20,"zIndex":28672,"backgroundColor":"transparent","fontWeight":"bold","fontSize":14,"verticalAlign":"middle","color":"#999999","textAlign":"center","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","borderWidth":0,"drawAs":"none","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Label","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Label"},"daafb":{"textContent":"Product","id":"daafb","wpType":"Metabolite","x":373.88250118128843,"y":170.6562713288181,"width":80,"height":20,"zIndex":32768,"fontSize":10,"verticalAlign":"middle","color":"#0000ff","textAlign":"center","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","Metabolite"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"ab8f5":{"id":"ab8f5","cellularComponent":"Mitochondria","x":486.99435057220853,"y":153.48837209302343,"width":224.8062015503869,"height":125.96899224806182,"zIndex":16384,"fontSize":10,"verticalAlign":"middle","drawAs":"Mitochondria","borderWidth":3,"color":"#c0c0c0","rotation":0,"textAlign":"center","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode","PhysicalEntity","CellularComponent","Mitochondria"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"dce23":{"textContent":"Label","id":"dce23","x":284,"y":210,"width":80,"height":20,"zIndex":28672,"backgroundColor":"#ffffff","fontSize":10,"verticalAlign":"middle","drawAs":"Ellipse","textAlign":"center","color":"#000000","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Label","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Label"},"c7084":{"textContent":"Catalyst","id":"c7084","wpType":"GeneProduct","citations":["publicationXreffbe"],"x":293.88250118128843,"y":118.6562713288181,"width":80,"height":20,"zIndex":32768,"fontSize":10,"verticalAlign":"middle","textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"b621a":{"id":"b621a","cellularComponent":"Nucleus","borderStyle":"double","x":483.7713178294573,"y":320.98062015503865,"width":204.5503875969004,"height":153.38759689922378,"zIndex":16384,"fontSize":10,"verticalAlign":"middle","drawAs":"Ellipse","borderWidth":3,"color":"#c0c0c0","rotation":0,"textAlign":"center","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode","PhysicalEntity","CellularComponent","Nucleus"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"b07f7":{"textContent":"Mitochondria","id":"b07f7","x":525.004572532328,"y":132.67864926540335,"width":144.7286821705431,"height":21.5503875968991,"zIndex":32769,"backgroundColor":"transparent","fontWeight":"bold","fontSize":14,"verticalAlign":"middle","color":"#999999","textAlign":"center","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","borderWidth":0,"drawAs":"none","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Label","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Label"},"d8bae":{"textContent":"TCA Cycle","id":"d8bae","wpType":"Pathway","x":558.6222575489527,"y":210.15503875968992,"width":80,"height":20,"zIndex":32768,"fontWeight":"bold","fontSize":12,"verticalAlign":"middle","drawAs":"none","color":"#14961e","textAlign":"center","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","borderWidth":0,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"dbConventionalName":"WikiPathways","dbId":"WP78","type":["DataNode","SingleFreeNode","Pathway"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"da9a8":{"textContent":"D-Glucose","id":"da9a8","wpType":"Metabolite","x":203.41085271317831,"y":307.82945736434135,"width":80,"height":20,"zIndex":32768,"fontSize":10,"verticalAlign":"middle","color":"#0000ff","textAlign":"center","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"dbConventionalName":"HMDB","dbId":"HMDB00122","type":["DataNode","SingleFreeNode","Metabolite"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"d0619":{"textContent":"APC","id":"d0619","wpType":"GeneProduct","x":12,"y":12,"width":55.2030544949669,"height":19.999999999999993,"zIndex":32768,"fontSize":10,"verticalAlign":"middle","textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"dbConventionalName":"Ensembl","dbId":"ENSG00000134982","type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode","isPartOf":"aecda"},"f376f":{"textContent":"TP53","id":"f376f","wpType":"GeneProduct","x":293.333333333333,"y":255.11627906976736,"width":80,"height":20,"zIndex":32768,"fontSize":10,"verticalAlign":"middle","textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"dbConventionalName":"Ensembl","dbId":"ENSG00000141510","type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"d32e4":{"textContent":"Isocitric acid","id":"d32e4","wpType":"Metabolite","x":513.4883720930228,"y":380.6976744186048,"width":80,"height":20,"zIndex":32768,"fontSize":10,"verticalAlign":"middle","color":"#0000ff","textAlign":"center","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"dbConventionalName":"HMDB","dbId":"HMDB00193","type":["DataNode","SingleFreeNode","Metabolite"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"f917f":{"textContent":"ADP","id":"f917f","wpType":"Metabolite","x":198,"y":347,"width":34,"height":15,"zIndex":32768,"fontSize":10,"verticalAlign":"middle","color":"#0000ff","textAlign":"center","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"dbConventionalName":"HMDB","dbId":"HMDB01341","type":["DataNode","SingleFreeNode","Metabolite"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"e9680":{"textContent":"GeneProduct","id":"e9680","wpType":"GeneProduct","x":384,"y":137,"width":80,"height":20,"zIndex":32768,"fontSize":10,"verticalAlign":"middle","textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"a80da":{"textContent":"CASP10","id":"a80da","wpType":"GeneProduct","x":12.000000000000085,"y":32,"width":55.20305449496689,"height":20,"zIndex":32770,"fontSize":10,"verticalAlign":"middle","textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"dbConventionalName":"Entrez Gene","dbId":"843","type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode","isPartOf":"aecda"},"f9e19":{"textContent":"Isocitric acid","id":"f9e19","wpType":"Metabolite","x":346.74418604651123,"y":381.3953488372092,"width":80,"height":20,"zIndex":32771,"fontSize":10,"verticalAlign":"middle","color":"#0000ff","textAlign":"center","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"dbConventionalName":"HMDB","dbId":"HMDB00193","type":["DataNode","SingleFreeNode","Metabolite"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"d6c00":{"textContent":"ATP","id":"d6c00","wpType":"Metabolite","x":338,"y":456,"width":34,"height":15,"zIndex":32772,"fontSize":10,"verticalAlign":"middle","color":"#0000ff","textAlign":"center","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"dbConventionalName":"HMDB","dbId":"HMDB01532","type":["DataNode","SingleFreeNode","Metabolite"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"publicationXrefbf7":{"id":"publicationXrefbf7","dbId":"18651794","dbConventionalName":"PubMed","standardName":"WikiPathways: pathway editing for the people.","source":"PLoS Biol","year":2008,"authors":["Pico AR","Kelder T","van Iersel MP","Hanspers K","Conklin BR","Evelo C"],"type":["PublicationXref"],"gpmlElementName":"PublicationXref","textContent":"1"},"publicationXrefb25":{"id":"publicationXrefb25","dbId":"19649250","dbConventionalName":"PubMed","standardName":"Mining biological pathways using WikiPathways web services.","source":"PLoS One","year":2009,"authors":["Kelder T","Pico AR","Hanspers K","van Iersel MP","Evelo C","Conklin BR"],"type":["PublicationXref"],"gpmlElementName":"PublicationXref","textContent":"2"},"publicationXreffbe":{"id":"publicationXreffbe","dbId":"20824171","dbConventionalName":"PubMed","standardName":"Finding the right questions: exploratory pathway analysis to enhance biological discovery in large datasets.","source":"PLoS Biol","year":2010,"authors":["Kelder T","Conklin BR","Evelo CT","Pico AR"],"type":["PublicationXref"],"gpmlElementName":"PublicationXref","textContent":"3"},"publicationXrefafa":{"id":"publicationXrefafa","dbId":"22096230","dbConventionalName":"PubMed","standardName":"WikiPathways: building research communities on biological pathways.","source":"Nucleic Acids Res","year":2012,"authors":["Kelder T","van Iersel MP","Hanspers K","Kutmon M","Conklin BR","Evelo CT","Pico AR"],"type":["PublicationXref"],"gpmlElementName":"PublicationXref","textContent":"4"},"http://identifiers.org/doid/DOID:344":{"term":"disease of biological process","dbId":"DOID:344","ontology":"Disease","type":["openControlledVocabulary"],"gpmlElementName":"openControlledVocabulary","id":"http://identifiers.org/doid/DOID:344"},"http://identifiers.org/pw/PW:0001310":{"term":"abscisic acid biosynthetic pathway","dbId":"PW:0001310","ontology":"Pathway Ontology","type":["openControlledVocabulary"],"gpmlElementName":"openControlledVocabulary","id":"http://identifiers.org/pw/PW:0001310"},"http://identifiers.org/pw/PW:0000125":{"term":"G protein mediated signaling pathway","dbId":"PW:0000125","ontology":"Pathway Ontology","type":["openControlledVocabulary"],"gpmlElementName":"openControlledVocabulary","id":"http://identifiers.org/pw/PW:0000125"},"idd31b36db":{"id":"idd31b36db","zIndex":12288,"borderWidth":1,"points":[{"x":293.88250118128843,"y":180.6562713288181,"isAttachedTo":"a489f","attachmentDisplay":{"position":[1,0.5],"offset":[0,0]},"orientation":[1,0]},{"x":373.88250118128843,"y":180.6562713288181,"isAttachedTo":"daafb","attachmentDisplay":{"position":[0,0.5],"offset":[0,0]},"orientation":[1,0]}],"color":"#000000","drawAs":"StraightLine","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","markerEnd":"MimConversion","isAttachedTo":["a489f","daafb"],"burrs":["ac1b7"]},"ac1b7":{"attachmentDisplay":{"position":[0.5,0],"offset":[0,0]},"drawAs":"none","id":"ac1b7","isAttachedTo":"idd31b36db","borderWidth":0,"height":0,"width":0,"zIndex":12288,"type":["Anchor","Burr"],"kaavioType":"Burr","gpmlElementName":"Anchor"},"id1b82938b":{"id":"id1b82938b","zIndex":12288,"borderWidth":1,"points":[{"x":333.88250118128843,"y":138.6562713288181,"isAttachedTo":"c7084","attachmentDisplay":{"position":[0.5,1],"offset":[0,0]},"orientation":[0,1]},{"x":333.88250118128843,"y":180.6562713288181,"isAttachedTo":"ac1b7","attachmentDisplay":{"position":[0.5,0],"offset":[],"relativeOffset":[0,0]},"orientation":[]}],"color":"#000000","drawAs":"StraightLine","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","markerEnd":"MimCatalysis","isAttachedTo":["c7084","idd31b36db"]},"iddb017374":{"id":"iddb017374","zIndex":12288,"strokeDasharray":"5,3","borderWidth":1,"points":[{"x":453.88250118128843,"y":180.6562713288181,"isAttachedTo":"daafb","attachmentDisplay":{"position":[1,0.5],"offset":[0,0]},"orientation":[1,0]},{"x":558.6222575489527,"y":220.15503875968992,"isAttachedTo":"d8bae","attachmentDisplay":{"position":[0,0.5],"offset":[0,0]},"orientation":[1,0]}],"color":"#000000","drawAs":"StraightLine","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","markerEnd":"Arrow","isAttachedTo":["daafb","d8bae"]},"id476a1b22":{"id":"id476a1b22","drawAs":"ElbowLine","zIndex":12288,"borderWidth":1,"points":[{"x":243.41085271317831,"y":307.82945736434135,"isAttachedTo":"da9a8","attachmentDisplay":{"position":[0.5,0],"offset":[0,0]},"orientation":[0,-1]},{"x":213.13399778516057,"y":243.4108527131783},{"x":182.85714285714283,"y":212.0335620209982},{"x":213.88250118128843,"y":180.6562713288181,"isAttachedTo":"a489f","attachmentDisplay":{"position":[0,0.5],"offset":[0,0]},"orientation":[1,0]}],"color":"#000000","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","markerEnd":"MimConversion","isAttachedTo":["da9a8","a489f"],"burrs":["d144a"]},"d144a":{"attachmentDisplay":{"position":[0.19253891532885647,0],"offset":[0,0]},"drawAs":"none","id":"d144a","isAttachedTo":"id476a1b22","borderWidth":0,"height":0,"width":0,"zIndex":12288,"type":["Anchor","Burr"],"kaavioType":"Burr","gpmlElementName":"Anchor"},"ida549ffdf":{"id":"ida549ffdf","citations":["publicationXrefafa"],"zIndex":12288,"borderWidth":1,"points":[{"x":262.2158476719392,"y":438.27662352688367,"isAttachedTo":"aecda","attachmentDisplay":{"position":[1,0.25],"offset":[0,0]},"orientation":[1,0]},{"x":294.957125723838,"y":415.5704732069022,"isAttachedTo":"fee36","attachmentDisplay":{"position":[0.45501286271416125,0],"offset":[],"relativeOffset":[0,0]},"orientation":[]}],"color":"#000000","drawAs":"StraightLine","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","markerEnd":"MimCatalysis","isAttachedTo":["aecda","ida7510884"]},"ida7510884":{"id":"ida7510884","drawAs":"CurvedLine","zIndex":12288,"borderWidth":1,"points":[{"x":366.74418604651123,"y":401.3953488372092,"isAttachedTo":"f9e19","attachmentDisplay":{"position":[0.25,1],"offset":[0,0]},"orientation":[0,1]},{"y":421.3953488372092,"x":315.0775193798448},{"x":263.4108527131783,"y":327.82945736434135,"isAttachedTo":"da9a8","attachmentDisplay":{"position":[0.75,1],"offset":[0,0]},"orientation":[0,-1]}],"color":"#000000","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","markerEnd":"MimConversion","isAttachedTo":["f9e19","da9a8"],"burrs":["fee36","ab8e9","baddb","add43"]},"fee36":{"attachmentDisplay":{"position":[0.45501286271416125,0],"offset":[0,0]},"drawAs":"none","id":"fee36","isAttachedTo":"ida7510884","borderWidth":0,"height":0,"width":0,"zIndex":12288,"type":["Anchor","Burr"],"kaavioType":"Burr","gpmlElementName":"Anchor"},"ab8e9":{"attachmentDisplay":{"position":[0.7141711819012988,0],"offset":[0,0]},"drawAs":"none","id":"ab8e9","isAttachedTo":"ida7510884","borderWidth":0,"height":0,"width":0,"zIndex":12288,"type":["Anchor","Burr"],"kaavioType":"Burr","gpmlElementName":"Anchor"},"baddb":{"attachmentDisplay":{"position":[0.12411762777416682,0],"offset":[0,0]},"drawAs":"none","id":"baddb","isAttachedTo":"ida7510884","borderWidth":0,"height":0,"width":0,"zIndex":12288,"type":["Anchor","Burr"],"kaavioType":"Burr","gpmlElementName":"Anchor"},"add43":{"attachmentDisplay":{"position":[0.6815968775798219,0],"offset":[0,0]},"drawAs":"none","id":"add43","isAttachedTo":"ida7510884","borderWidth":0,"height":0,"width":0,"zIndex":12288,"type":["Anchor","Burr"],"kaavioType":"Burr","gpmlElementName":"Anchor"},"id1006ca65":{"id":"id1006ca65","zIndex":12288,"borderWidth":1,"points":[{"x":293.333333333333,"y":265.11627906976736,"isAttachedTo":"f376f","attachmentDisplay":{"position":[0,0.5],"offset":[0,0]},"orientation":[-1,0]},{"x":243.41085271317831,"y":265.71113561190737,"isAttachedTo":"d144a","attachmentDisplay":{"position":[0.19253891532885647,0],"offset":[],"relativeOffset":[0,0]},"orientation":[]}],"color":"#000000","drawAs":"StraightLine","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","markerEnd":"MimCatalysis","isAttachedTo":["f376f","id476a1b22"]},"id3a237139":{"id":"id3a237139","zIndex":12288,"borderWidth":1,"points":[{"x":513.4883720930228,"y":390.6976744186048,"isAttachedTo":"d32e4","attachmentDisplay":{"position":[0,0.5],"offset":[0,0]},"orientation":[-1,0]},{"x":426.74418604651123,"y":391.3953488372092,"isAttachedTo":"f9e19","attachmentDisplay":{"position":[1,0.5],"offset":[0,0]},"orientation":[-1,0]}],"color":"#000000","drawAs":"StraightLine","dbConventionalName":"Rhea","dbId":"10003","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","markerEnd":"MimConversion","isAttachedTo":["d32e4","f9e19"]},"id55ba47f0":{"id":"id55ba47f0","drawAs":"CurvedLine","zIndex":12288,"borderWidth":1,"points":[{"x":272.5889076292645,"y":383.16222959363637,"isAttachedTo":"add43","attachmentDisplay":{"position":[0.6815968775798219,0],"offset":[],"relativeOffset":[0,0]},"orientation":[-1,0]},{"y":368.83111479681816,"x":252.5889076292645},{"x":232,"y":354.5,"isAttachedTo":"f917f","attachmentDisplay":{"position":[1,0.5],"offset":[0,0]},"orientation":[-1,0]}],"color":"#000000","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","markerEnd":"MimConversion","isAttachedTo":["ida7510884","f917f"]},"idd937525f":{"id":"idd937525f","zIndex":12288,"borderWidth":1,"points":[{"x":346.5,"y":456,"isAttachedTo":"d6c00","attachmentDisplay":{"position":[0.25,0],"offset":[0,0]},"orientation":[0,-1]},{"x":351.7875185532506,"y":416.04611755912117,"isAttachedTo":"baddb","attachmentDisplay":{"position":[0.12411762777416682,0],"offset":[],"relativeOffset":[0,0]},"orientation":[]}],"color":"#000000","drawAs":"StraightLine","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","markerEnd":"MimConversion","isAttachedTo":["d6c00","ida7510884"]},"aecda":{"id":"aecda","citations":["publicationXrefb25"],"textAlign":"center","color":"#808080","verticalAlign":"middle","fontSize":1,"fontWeight":"bold","borderWidth":1,"fillOpacity":0.1,"padding":11,"backgroundColor":"#b4b464","type":["Group","Complex","GroupComplex"],"contains":["d0619","a80da"],"kaavioType":"Group","gpmlElementName":"Group","zIndex":32768,"x":183.01279317697222,"y":422.27662352688367,"width":79.20305449496698,"height":64}}}
